modules/ROOT/pages/release-notes.adoc

```diff
@@ -171,9 +171,43 @@ Bonita Runtime produces a warning log if the filter is set and ignores it.
 
 The tenant creation method has been deprecated following our 2021.1 decision to deprecate the Multi-Tenants architecture.
 
+=== Potential need of declaring external Maven Repository
+
+From version 2022.1-u2, if you embed *bonita-server(-sp).jar* in your application, you need to declare additional Maven Repository in your pom.xml / build.gradle. This is due to Security Fixes that required the use of a dependency NOT present in default Maven Repository (aka Maven Central). To do so, add the following code:
```

Comment:
🔥 Adding repositories in pom.xml has been described as a bad practise for years
- https://blog.sonatype.com/2009/02/why-putting-repositories-in-your-poms-is-a-bad-idea/ (2009)
- https://maven.apache.org/repository-management.html

The remark here should at least advise usage of repository manager. This is not the only place where this should advice!